modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -216,12 +216,24 @@ export default class MVTLayer extends TileLayer {
     return renderedFeatures;
   }
 
+  async getVisibleTiles(tileCoords = 'local') {
```

Comment:
Why is `tileCoords` required? You can just deduce the coordinate system from information in this layer.

Reply:
Let's say `resolution` is = `'local'`, but user needs the tile data in WGS84 :roll_eyes: